ims/_journals.tab

```diff
@@ -0,0 +1,6 @@
+Title	Title-short	ISSN	Documentation
+The Annals of Applied Probability	AAP	1050-5164	http://www.imstat.org/aap/manprep.html
+The Annals of Probability	AOP	 	0091-1798	http://www.imstat.org/aop/manprep.htm
+The Annals of Statistics	AOS	 	0090-5364	http://www.imstat.org/aos/manprep.html
+Statistical Science	STS	x	http://www.imstat.org/sts/manprep.html
+The Electronic Journal of Statistics	EJS	1935-7524	http://www.imstat.org/ejs/mansub.html
```

Comment:
That's the ones I checked and think that they should follow the style I made. With those LaTeX citationstyles (or whatever they are) I am unsure now though and would like to check this.